frontend/src/components/forms/_elements/form-wrapper/FormWrapper.tsx

```diff
@@ -1,21 +1,24 @@
 import React from 'react'
 import {useAppDispatch, useAppSelector} from "../../../../app/hooks";
-
+import {
+    createSupplier,
+    selectSupplierToSave,
+    validateSupplier
+} from "../../../../slicer/supplierSlice";
+import {toBeReplaced} from "../../../../slicer/employeeSlice";
+import {createProduct, selectProductToSave, validateProduct} from "../../../../slicer/productSlice";
+import {createOrder, selectOrderToSave, validateOrder} from "../../../../slicer/orderSlice";
```

Comment:
maybe [this is a good way](https://stackoverflow.com/questions/34925992/how-to-avoid-imports-with-very-long-relative-paths-in-angular-2) to avoid those long import paths

Reply:
good to know, on the other side all these import paths get autogenerated....